content/en/docs/faq/installation/uninstall-pluggable-components.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "Uninstall Pluggable Components from KubeSphere v3.1.x"
+keywords: "Installer, uninstall, KubeSphere, Kubernetes"
+description: "Learn how to uninstall each pluggable component in KubeSphere v3.1.x."
+linkTitle: "Uninstall Pluggable Components from KubeSphere v3.1.x"
+Weight: 16500
+---
+
+After you [enable the pluggable components of KubeSphere](../../../pluggable-components/), you can also uninstall them by performing the following steps. Please back up any necessary data before you uninstall these components.
+
+{{< notice note >}}
+
+The methods of uninstalling certain pluggable components on KubeSphere v3.1.x are different from the methods on KubeSphere v3.0.0. For more information about the uninstallation methods on KubeSphere v3.0.0, see [Uninstall Pluggable Components from KubeSphere](https://v3-0.docs.kubesphere.io/docs/faq/installation/uninstall-pluggable-components/).
+
+{{</ notice >}}
+
+## Prerequisites
+
+You have to change the value of the field `enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration` before you uninstall any pluggable components except Service Topology and Pod IP Pools. 
+
+Use either of the following methods to change the value of the field `enabled`:
+
+- Run the following command to edit `ks-installer`:
+
+  ```bash
+  kubectl -n kubesphere-system edit clusterconfiguration ks-installer
+  ```
+
+- Log in to the KubeSphere web console as `admin`, click **Platform** in the upper-left corner and select **Cluster Management**, and then go to **CRDs** to search for `ClusterConfiguration`. For more information, see [Enable Pluggable Components](../../../pluggable-components/).
+
+{{< notice note >}}
+
+After the value is changed, you need to wait until the updating process is complete before you continue with any further operations.
+
+{{</ notice >}}
+
+## Uninstall KubeSphere App Store
+
+Change the value of `openpitrix.store.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
+
+## Uninstall KubeSphere DevOps
+
+1. Change the value of `devops.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
```

Comment:
How about deleting `status.devops` at the same time?

Reply:
Thanks for your comment. I'm not sure if this is required, but I'm with you if you are sure about this. So, where can I delete `status.devops`?

Reply:
Hum... I just found out that only part of the YAML file of `ks-installer` is shown on the console. The `status.devops` part is not shown.
<img width="1234" alt="Xnip2021-08-20_09-49-13" src="https://user-images.githubusercontent.com/75110798/130166454-6bf2cead-3a93-40a7-afe4-48bded12bbbf.png">

But I can see that part on CLI. Additionally, if DevOps needs to delete the `status.devops` part, will that apply to other components? 
<img width="839" alt="Xnip2021-08-20_09-53-38" src="https://user-images.githubusercontent.com/75110798/130166695-3fdf66ab-e68d-4c5f-89c0-7348f9a98953.png">

Reply:
If the status is not deleted, users cannot enable DevOps again by setting devops.enable to true.

Reply:
Sure. I'll update the doc.

Reply:
> Hum... I just found out that only part of the YAML file of `ks-installer` is shown on the console. The `status.devops` part is not shown.
> <img alt="Xnip2021-08-20_09-49-13" width="1234" src="https://user-images.githubusercontent.com/75110798/130166454-6bf2cead-3a93-40a7-afe4-48bded12bbbf.png">
> 
> But I can see that part on CLI. Additionally, if DevOps needs to delete the `status.devops` part, will that apply to other components?
> ![Xnip2021-08-20_09-53-38](https://user-images.githubusercontent.com/75110798/130167662-edec2d59-f56f-4008-b1fb-2b02c13e229f.png)

Hi @zryfish, I'm thinking if the DevOps component requires the deletion of `status.devops`, do other components also need to do delete their corresponding part?

Reply:
> Hum... I just found out that only part of the YAML file of `ks-installer` is shown on the console. The `status.devops` part is not shown.
> <img alt="Xnip2021-08-20_09-49-13" width="1234" src="https://user-images.githubusercontent.com/75110798/130166454-6bf2cead-3a93-40a7-afe4-48bded12bbbf.png">
> 
> But I can see that part on CLI. Additionally, if DevOps needs to delete the `status.devops` part, will that apply to other components?
> ![Xnip2021-08-20_09-53-38](https://user-images.githubusercontent.com/75110798/130167662-edec2d59-f56f-4008-b1fb-2b02c13e229f.png)

Since other components did not mention anything about the deletion of `status.XXX`, I'll take it that other components do not require this operation. Please let me know if there's any further updates.

Reply:
Since other components did not mention anything about the deletion of `status.XXX`, I'll take it that other components do not require this operation. Please let me know if there are any further updates.